server/src/sim/financialanalysis.py

```diff
@@ -48,7 +47,7 @@ def financialanalysis(D, postyear=2015, S=None, rerunmodel=False, artgrowthrate=
 
     # Get most recent ART unit costs 
     progname = 'ART'
-    prognumber = D['data']['meta']['progs']['short'].index(progname)
```

Comment:
@robynstuart if you can think of a way of updating D['programs'] before any calculation takes place, then it should not be a problem. Currently D['programs'] is only updated during loading data from spreadsheet (first time):
```
    if 'programs' not in D:
        D['programs'] = restructureprograms(programs)
```